features/tests/e2e/steps/transactions-steps.js

```diff
@@ -28,18 +28,14 @@ Given(
           new window.Promise(resolve =>
             // Need to fetch the wallets data async and wait for all results
             window.Promise.all([
```

Comment:
@tomothespian I don't think we need `Promise.all` here anymore...

Reply:
@nikolaglumac you're in right 👍

Reply:
@nikolaglumac Done, can you re-check please?